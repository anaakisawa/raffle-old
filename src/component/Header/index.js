import React, { Component } from 'react';

class Header extends Component {
  render() {
    return(
      <div className="Header">
        <p>{this.props.text}</p>
      </div>
    );
  }
}

export default Header;
