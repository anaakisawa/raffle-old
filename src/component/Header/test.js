
import React from 'react';
import Header from './index';
import renderer from 'react-test-renderer';

describe('Component - Header', () => {
  test('renders a text element', () => {
    const component = renderer.create(
      <Header text="it works"/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
