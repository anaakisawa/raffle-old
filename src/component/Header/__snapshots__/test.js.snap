// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Header renders a text element 1`] = `
<div
  className="Header"
>
  it works
</div>
`;
